docs/ops/arithmetic/Floor_1.md

```diff
@@ -22,11 +29,6 @@
 
 * *T*: any numeric type.
```

Comment:
Please add \*T* (stars) to the T in the input/utput description. Also we usually add shape info. 
And there should be "A tensor" instead of "An tensor".

* **1**: An tensor of type T. **Required.** 
->
* **1**: A tensor of type *T* and arbitrary shape. **Required.**

Reply:
done